### PR TITLE
new readme and related screenshots

### PR DESCRIPTION
**Pull request by spediciniPTC**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,73 +1,161 @@
...
-## References
-* [Onshape Developer Portal](https://dev-portal.onshape.com)
-    * [Help / Documentation](https://dev-portal.onshape.com/help)
-* [Heroku](https://heroku.com)
-    * [Heroku CLI](https://devcenter.heroku.com/articles/heroku-cli)
-* [GLTF information](https://www.khronos.org/gltf/) from Khronos Group
-* [three.js](https://threejs.org/) library to render GLTF
-                     
+## **README**
+The GLTF Viewer is a sample application that demonstrates how to **deploy an Onshape application**. It enables visualization of GLTF data translated from an Onshape model. It is a Node.JS application that runs as a tab inside an Onshape document. Onshape passes the document context to the viewer, which is used to help generate the GLTF visualization.
```
Comment (PaulJPremakumar) [inline README.md:2]: I would keep the purpose as follows.- "The GLTF Viewer is a sample application that demonstrates how to fetch a glTF representation of an Onshape model. "
Could we also add ?

Also demonstrated
* Creating an app that runs as a tab inside an onshape document
* OAuth - Authentication
* Use of REST APIs
* Use of document context

The application is built using Express and deployed on Heroku.

While we do demonstrate how to deploy, that isn't the main goal of the app, nor do go into details.

File: README.md [hunk 2]
```diff
@@ -1,73 +1,161 @@
...
+
+## Prerequisites
+
+1. Install the necessary software:  
+    * Git
+    * npm
+    * Heroku CLI (or the deployment server of your choosing)
+      * MacOS: `brew tap heroku/brew && brew install heroku` 
+      * Linux distributions that support Snap applications: `sudo snap install --classic heroku`
+      * Windows: Coming soon
```
Comment (PaulJPremakumar) [inline README.md:16]: We should take this out. It has been there for a while.

File: README.md [hunk 3]
```diff
@@ -1,73 +1,161 @@
...
+    * Docker
```
Comment (PaulJPremakumar) [inline README.md:17]: I am not sure we need Docker

**Review by PaulJPremakumar — APPROVED** [review]
Looks good. Hopefully in a week, we should have the app Brett worked on deployed in Onshape, we should direct users to try the app out in the documentation. But we don't need to wait for it.

**Review by pbhide17 — APPROVED** [review]

**Review by btorra-onshape — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (spediciniPTC) [reply README.md:17]: Same. I got Heroku errors when Docker was closed.

File: README.md (hunk 2 above)
Comment (spediciniPTC) [reply README.md:16]: @PaulJPremakumar I haven't had a chance to walk through the entire steps with Windows yet. Once I do that I will update this README with the windows instructions.

**Comment by spediciniPTC** [comment]
Will merge once QA walks through the steps

**Review by jramsley — APPROVED** [review]

**Pull request merged**